lib/ts/recipe/session/index.ts

```diff
@@ -18,7 +18,7 @@ import { RecipeInterface } from "supertokens-website";
 import SessionAuthWrapper from "./sessionAuth";
 import useSessionContextFunc from "./useSessionContext";
 import { InputType, SessionContextType } from "./types";
-import SessionContext from "./sessionContext";
+import SessionContext from "./SessionContext";
```

Comment:
IMPORTANT: I had renamed this file to be `sessionContext`. The capital letter at the start causes issues with Github since github  seems to ignore capital letter starts to the name of the file..